classes/requests/helpers/class-dintero-checkout-helper-base.php

```diff
@@ -23,6 +23,35 @@ public static function format_number( $number ) {
 		return intval( number_format( $number * 100, 0, '', '' ) );
 	}
 
+	/**
+	 * Retrieve the shipping method option for a given setting name.
+	 *
+	 * @param WC_Shipping_Rate|WC_Order_Item_Shipping $shipping The shipping rate or order item shipping to retrieve from.
+	 * @param string                                  $setting_name The setting name to retrieve. The prefix 'dintero_' can be omitted.
+	 * @return mixed The setting value or null if not found.
```

Comment:
Suggestion: Instead of mixed as a return type, set it to be `string|null`. That will make it a bit more clear when sing this function and will help with static code analysis to catch possible errors. Or simply change it to return an empty string when we don't have a description.